doc/faq.rst

```diff
@@ -0,0 +1,283 @@
+
+   * ***Windows Only***: Launch it via the Start menu shortcut.
+
+     .. image:: images/faq/faq-windows-launch.png
+        :alt: Navigator showing running a specific version of Spyder
+
+
+.. dropdown:: Q: Can I try Spyder without installing it?
+
+   Yes!
+   With `Binder`_ you can work with a fully functional copy of Spyder online that runs right in your web browser.
+   Try it `here`_.
+
+   .. _Binder: https://mybinder.org/
+   .. _here: https://mybinder.org/v2/gh/spyder-ide/spyder/4.x?urlpath=/desktop
+
+
+.. dropdown:: Q: What are the system requirements for Spyder? How resource-intensive is it?
+
+   Spyder works on modern versions of Windows, macOS and Linux (see table below for recommended versions) via Anaconda, as well as other methods.
+   It typically uses relatively minimal CPU when idle, and 0.5 GB - 1 GB of RAM, depending on how long you've been using it and how many files, projects, panes and consoles you have open.
+   It should work on any system with a dual-core or better x64 processor and at least 4 GB of RAM, although 8 GB is strongly recommended for best performance when running other applications.
+   However, the code you run, such as scientific computation and deep learning models, may require additional resources beyond this baseline for Spyder itself.
+
+   .. table::
+
+      ================   ===================
+      Operative system   Version
+      ================   ===================
+      Windows            Windows 8
+      macOS              High Sierra (10.13)
+      Linux              Ubuntu 16.04
+      ================   ===================
+
+
+.. dropdown:: Q: How do I run Spyder in a conda environment using Anaconda Navigator?
+
+   Select the environment you want to launch Spyder from under :guilabel:`Applications on`.
+   If Spyder is installed in this environment, you will see it on Navigator's :guilabel:`Home` window.
+   Click :guilabel:`Launch` to use Spyder with your selected environment.
+
+   .. image:: images/faq/faq-run-environment.png
+      :alt: Navigator showing running Spyder in a specific environment
+
+
+.. dropdown:: Q: How do I run Spyder in a conda environment using the command line?
+
+   Activate your conda environment typing on your command line (or Anaconda Prompt on Windows):
+
+   .. code-block:: bash
+
+      conda activate <ENVIRONMENT-NAME>
+
+   Then, type ``spyder``.
+
+
+
+============
+Using Spyder
+============
+
+.. dropdown:: Q: How do I install Python packages to use within Spyder?
+
+   The first approach for installing a package should be using conda.
+   In your command line (or Anaconda Prompt on Windows) type:
+
+   .. code-block:: bash
+
+      conda install <PACKAGE-NAME>
+
+   If your installation is not successful go through steps 3 to 5 of Part 2 in our `video`_ on solving and avoiding problems with pip, Conda and Conda-Forge.
+
+   .. _video: https://www.youtube.com/watch?v=Ul79ihg41Rs&t=306s
```

Comment:
Not sure about this. Let's see what @ccordoba12 thinks

Reply:
Since the sections are all collapsible, I'm not sure what possible downside you see to embedding it directly so users can view it more easily, rather than having to click through a link (which can sharply reduce the rate at which they do so and takes them away from our site), but we'll see what Carlos thinks.

Reply:
I agree with @CAM-Gerlach. Let's embed it here and make it start at the right time.

Reply:
Implemented